src/patternfly/components/Tabs/examples/Tabs.md

```diff
@@ -50,6 +50,20 @@ import './Tabs.css'
 | `.pf-m-scrollable` | `.pf-c-tabs` | Enables the directional scroll buttons. |
 | `.pf-c-tabs__scroll-button` | `<button>` | Initiates a tabs component scroll button. |
 
+### Horizontal overflow
+```hbs isBeta
+{{#> tabs tabs--id="horizontal-overflow-example"}}
+  {{> __tabs-list __tabs-list--NoScrollButtons="true" __tabs-list--IsOverflow="true"}}
+{{/tabs}}
+```
+
+### Horizontal overflow expanded
+```hbs isBeta
+{{#> tabs tabs--id="horizontal-overflow-expanded-example"}}
+  {{> __tabs-list __tabs-list--NoScrollButtons="true" __tabs-list--IsOverflow="true" __tabs-list--IsOverflowExpanded="true"}}
+{{/tabs}}
+```
+
```

Comment:
Can you also include an example of what it looks like when the overflow tab is selected (the result of selecting an item from the menu). From the design:

<img width="1102" alt="Screen Shot 2022-05-27 at 11 38 45 AM" src="https://user-images.githubusercontent.com/35148959/170742019-c86c4e0c-b9bb-4054-bf5f-863a02d7851d.png">